src/main/kotlin/org/rust/ide/intentions/RsShowMacroExpansionIntentions.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.parentOfType
+import org.rust.ide.actions.macroExpansion.MacroExpansionViewCreator
+import org.rust.lang.core.psi.RsMacroCall
+
+abstract class RsShowMacroExpansionIntentionBase(expandRecursively: Boolean) : RsElementBaseIntentionAction<RsMacroCall>() {
```

Comment:
@vlad20012 @Undin Should I use constructor parameter, or is it better to make abstract/open property and override it in the inheritors, like in `SetMutableIntention`?